externs/olx.js

```diff
@@ -2606,6 +2607,14 @@ olx.interaction.SelectOptions.prototype.toggleCondition;
  */
 olx.interaction.SelectOptions.prototype.multi;
 
+/**
+ * A function that takes an {@link ol.Feature} and an {ol.layer.Layer} and
+ * returns true if the feature may be selected or false otherwise.
+ * @typedef {function(ol.Feature, ol.layer.Layer): boolean}
```

Comment:
`@typedef` - > `@type`.  And why don't you use `ol.interaction.SelectFilterFunction` here too?

We don't have a typedef for the layer filter so I think I'd drop the typedef entirely.